{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -12,22 +12,18 @@ license = "{{cookiecutter.license}}"
 readme = "README.md"
 include = ["README.md", "src/{{cookiecutter.__project_slug}}/schema", "project"]
 
-requires-python = ">=3.9"
+requires-python = ">=3.9,<4.0"
 
 dynamic = ["version"]
 
 dependencies = [
-  "linkml-runtime >=1.8.0",
+  "linkml-runtime >=1.1.24",
 ]
 
 [tool.poetry]
 requires-poetry = ">=2.0"
 version = "0.0.0"
 
-[tool.poetry.dependencies]
```

Comment:
Removing this essentially removes poetry < 2 support, right?

Reply:
Yes. Did you see the tip in #129 to have both version installed side-by-side?

Reply:
@matentzn Do you think this is something we're ready to do as a community?

Reply:
@amc-corey-cox i have no helpful opinion on the matter. ;)